Tests/LibCore/TestLibCoreMimeType.cpp

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2025, Andreas Kling <andreas@ladybird.org>
```

Comment:
Andreas didn't write this test, you did :)

Feel free to put your name+email. Other contributors have opted to use a more generic "the Ladybird Developers" copyright line.